src/main/java/com/redislabs/modules/rejson/JReJSON.java

```diff
@@ -217,6 +221,46 @@ public Long del(String key, Path path) {
     	return gson.fromJson(rep, clazz);
     }
 
+    /**
+     * Returns the documents from multiple key s. Non-existing keys are reported as
+     * null.
+     * 
+     * @param <T>   target class to serialize results
+     * @param clazz target class to serialize results
+     * @param keys  keys for the JSON documents
+     * @return a List of documents rooted at path
+     */
+    public <T> List<T> mget(Class<T> clazz, String... keys) {
+        return mget(Path.ROOT_PATH, clazz, keys);
+    }
+
+    /**
+     * Returns the values at path from multiple key s. Non-existing keys and
+     * non-existing paths are reported as null.
+     * 
+     * @param path  common path across all documents to root the results on
+     * @param <T>   target class to serialize results
+     * @param clazz target class to serialize results
+     * @param keys  keys for the JSON documents
+     * @return a List of documents rooted at path
+     */
+    public <T> List<T> mget(Path path, Class<T> clazz, String... keys) {
+        String[] args = Stream //
+                .of(keys, new String[] { path.toString() }) //
+                .flatMap(Stream::of) //
+                .toArray(String[]::new);
+
+        List<String> rep = Collections.emptyList();
```

Comment:
This should be okay.
```suggestion
        List<String> rep;
```